oidc-consumer/test/index.test.ts

```diff
@@ -0,0 +1,112 @@
+import { Firestore } from '@google-cloud/firestore';
```

Comment:
can use firebase-emulator for automated tests

Reply:
As per discussion, moved to next update / separate PR